tests/classes/test_client.php

```diff
@@ -108,6 +108,7 @@ public function get_maximum_upload_size() {
     public function support_signed_urls() {
         return false;
     }
+
     public function generate_signed_url($contenthash) {
         return 'Not supported';
```

Comment:
Not quite related to your patch but we should change this to throw a 'not implemented' exception ?

Reply:
Oh, thank you for pointing on that. We use `generate_presigned_url()` method and it throws exception. `generate_signed_url()` was the first version of this method and not used anymore. I will remove it in further commit.